plutus-use-cases/test/Spec/GameStateMachine.hs

```diff
@@ -280,7 +278,7 @@ noLockedFunds = do
     w      <- forAllQ $ elementsQ wallets
     secret <- viewContractState currentSecret
     val    <- viewContractState gameValue
-    when (val > Ada.getLovelace Ledger.minAdaTxOut) $ do
+    when (val >= Ada.getLovelace Ledger.minAdaTxOut) $ do
```

Comment:
I'm guessing we can't remove `minAdaTxOut` from here?

Reply:
I think to do it later as a next step that removes `minAdaTxOut`.